talk/owt/sdk/include/cpp/owt/p2p/p2pclient.h

```diff
@@ -64,7 +64,7 @@ class OWT_EXPORT P2PClientObserver {
    */
   virtual void OnServerDisconnected(){}
 
-#ifdef OWT_CLOUD_GAMING
+#ifdef OWT_CG_SERVER
```

Comment:
That's backward incompatible uAPI change. Not good.

Reply:
Changed to `#if defined(OWT_CG_SERVER) || defined(OWT_CLOUD_GAMING)`. `OWT_CLOUD_GAMING` will be removed later.